### PR TITLE
More changes to the website and tutorials

### PR DESCRIPTION
**Pull request by SHZ66**

### CODE REVIEW

**Comment by SHZ66** [comment]
Hopefully the `:suppress` macro will only disable line numbers but still show the figures

File: tutorials/aanm/calculations.rst [hunk 1]
```diff
@@ -71,8 +71,43 @@ are set to those of `tmap`, so that we can verify the result by checking the RMS
...
+
+   N_MODES = []
+
+   def callback(**kwargs):
+      modes = kwargs.pop('modes')
+      
+      N_MODES.append(len(modes))
+
+Note that **N_MODES** needs to be defined outside the function, at a global scope, in order 
+to save the value for each iteration. **modes** is a :class:`.ModeSet` object that give you
```
Comment (jamesmkrieger) [inline tutorials/aanm/calculations.rst:97]: gives with an s

File: tutorials/aanm/calculations.rst [hunk 2]
```diff
@@ -71,8 +71,43 @@ are set to those of `tmap`, so that we can verify the result by checking the RMS
...
+   N_MODES = []
+
+   def callback(**kwargs):
+      modes = kwargs.pop('modes')
+      
+      N_MODES.append(len(modes))
+
+Note that **N_MODES** needs to be defined outside the function, at a global scope, in order 
+to save the value for each iteration. **modes** is a :class:`.ModeSet` object that give you 
+the mode selected for deform the structure in an iteration. You have the access to all the
```
Comment (jamesmkrieger) [inline tutorials/aanm/calculations.rst:98]: mode(s)

**Review by jamesmkrieger — CHANGES_REQUESTED** [review]
Looks good. Just two grammatical things

**Comment by jamesmkrieger** [comment]
> Hopefully the `:suppress` macro will only disable line numbers but still show the figures

We could just keep the line numbers to be on the safe side.

**Comment by jamesmkrieger** [comment]
I'll merge and make these changes myself and build the website again

**Pull request merged**